charts/dex/templates/service.yaml

```diff
@@ -32,6 +32,9 @@ spec:
 {{- end }}
 {{- if hasKey .Values.service "loadBalancerIP" }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+{{- end }}
+{{- if hasKey .Values.service "clusterIP" }}
+  clusterIP: {{ .Values.service.clusterIP | quote }}
```

Comment:
That's a manual fix on a vendored chart. It deserves a dedicated commit, explaining what is done (add support for cluster IP)